docs/dev-guide/integration/events/index.rst

```diff
@@ -9,15 +9,46 @@ status. This is enabled by setting ``event_notifications_enabled`` under
 ``event_notification_url`` may be set if the notification AMQP server is not
 running locally on port 5672.
 
+.. warning::
+   It is highly recommended to use a version of ``python-kombu`` that includes
+   a `fix <https://github.com/celery/kombu/issues/455>`_ for a connection leak.
+   ``python-kombu-3.0.24-7.pulp`` and newer are known to include this fix.
```

Comment:
I think we should suggest `python-kombu-3.0.24.6.pulp` instead because that was the first version with the fix.

http://koji.katello.org/koji/buildinfo?buildID=20255